src/components/AgenciesPage/types/Agencies.ts

```diff
@@ -0,0 +1,14 @@
+export interface Agency {
+  id: number;
+  name: string;
+  type: string;
+  administrator: string;
+  country_code: string;
+  launchers: string;
+  image_url: string;
+  spacecraft?: string;
+  [key: string]: string | undefined | number;
+}
+
+export type AgencyItemContent = Omit<Agency, 'id' | 'image_url'>;
+export type AgenciesSpecification = Omit<Agency, 'id' | 'image_url' | 'name'>;
```

Comment:
Why do you use Omit here?

Reply:
I can use the interface and then extend it. But for me, it is more readable.
I can change it if it needed